nexum_core/benches/README.md

```
@@ -4,50 +4,48 @@ This directory contains comprehensive performance benchmarks for the `nexum_core
 
 ## Benchmark Categories
 
-### 1. Storage Engine Benchmarks (`storage_bench.rs`)
+### 1. Database Comparison Benchmarks (`db_comparison.rs`)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)